client/src/pages/Assign Badge/AccountHolderForm.jsx

```diff
@@ -100,36 +44,54 @@ export default function AccountHolderForm() {
     console.log("account holder submitting", accountValue, evidenceValue);
   };
 
+  const handleSubmit = values => {
+    alert(JSON.stringify(values, null, 2));
+  };
+
+  const formik = useFormik({
+    initialValues: {
+      accountHolder: "",
+      evidence: "",
+    },
+    onSubmit: values => {
+      handleSubmit(values);
+    },
+  });
+
   return (
-    <StyledForm
-      id = "create-badge-form"
-      {...formItemLayout}
-      layout={formLayout}
-      form={form}
-      initialValues={{
-        layout: formLayout,
-      }}
-      onValuesChange={onFormLayoutChange}
-    >
-      <Form.Item label="Select a FightPandemics account holder">
-        <SearchInput placeholder="Start typing names of the person you want to award badges to" forminputs={formInputs({ value })}/>
-      </Form.Item>
-      <Form.Item label="">
-        <span>Evidence</span>
-        <TextArea placeholder="Optionally provide evidence for the person who completed the task" forminputs={formInputs({ value })}/>
-      </Form.Item>
-      <Form.Item>
-        <PrimaryFormButton className="btn-right" type="primary" onClick={toggle} onSubmit={submitHandler()}>Assign Badge</PrimaryFormButton>
-        <Modal
-          isShowing={isShowing}
-          hide={toggle}
-          modalWidth={800}
-          title="Congratulations your badge has been assigned!"
-          buttonPrimary="Copy URL"
-          buttonSecondary="Link"
-          modalBodyText="Your badge has been sent to [recipient name]! They will receieve an email notification for their new badge. You now have xxx badges remaining."
-        />
-      </Form.Item>
-    </StyledForm>
+    <FormikProvider value={formik}>
+      <StyledForm
+        id = "create-badge-form"
+        {...formItemLayout}
+        layout={formLayout}
+        form={form}
+        initialValues={{
+          layout: formLayout,
+        }}
+        onValuesChange={onFormLayoutChange}
+      >
+        <FormikAntdForm>
+          <Form.Item label="Select a FightPandemics account holder">
+            <SearchInput name="accountHolder" placeholder="Start typing names of the person you want to award badges to" forminputs={formInputs({ value })}/>
+          </Form.Item>
+          <Form.Item label="">
+            <span>Evidence</span>
+            <TextArea name="evidence" placeholder="Optionally provide evidence for the person who completed the task" forminputs={formInputs({ value })}/>
+          </Form.Item>
+          <Form.Item>
+            <PrimaryFormButton className="btn-right" type="primary" onClick={toggle} onSubmit={handleSubmit}>Assign Badge</PrimaryFormButton>
```

Comment:
You will need to include the prop `htmlType="submit"` to hook up the form values to the Formik `onSubmit` property. See Antd [Button](https://ant.design/components/button/#API) API reference and [useFormik](https://formik.org/docs/api/useFormik) reference.